backend/settings.py

```diff
@@ -155,6 +154,15 @@
     }
 }
 
+# This is needed to allow the LTI tool to work with the Django cache when running on multiple workers
+CACHES = {
+    "default": {
+        "BACKEND": "django_mysql.cache.MySQLCache",
+        "LOCATION": "django_clrt_cache",
+        "KEY_PREFIX": "clrt",
+    }
+}
```

Comment:
Enabling the Mysql caching. https://django-mysql.readthedocs.io/en/latest/cache.html